practical/matlab/idr0021_steps.m

```diff
@@ -0,0 +1,203 @@
+% Note: Do not run the whole script. Select the code block
+% of each exercise, right-click and "Evaluate Selection". 
+% Then proceed to the next exercise.
+% The exercises build on top of each other (later exercises
```

Comment:
>The exercises build on top of each other

Remove this sentence, remove the bracket, leave the sentence starting as "Leave..." as the main statement